README.md

```diff
@@ -144,9 +144,6 @@ go generate ./...
 
 ## Release
 To release a version to the [Terraform Public Registry](https://registry.terraform.io/providers/env0/env0/latest?pollNotifications=true) -
-1. Docs must be generated manually and committed to the repo before release. (`./generate-docs.sh`)
-2. Create and push a tag locally, in semver format - `git tag v0.0.9 && git push origin --tags`
-3. Goto [Github Releases](https://github.com/env0/terraform-provider-env0/releases) and edit the draft created by Release Drafter Bot - it should contain the change log for the release. Make sure it's pointing at the tag you created in the previous step and publish the release.
-4. Binaries will be automatically generated by the Github action defined in `.github/workflows/release.yml`
-5. The Registry will automatically pick up on the new version. If the registry says the release doesn't have any binaries, check the result of the `release` Github action. You might need to [Resync](https://registry.terraform.io/providers/env0/env0/latest/settings?pollNotifications=true) the registry after the action finishes.
-
+1. Create and push a tag **locally**, in semver format - `git tag v0.0.9 && git push origin --tags`
+2. New release with binaries **will be automatically generated** by the GitHub action defined in `.github/workflows/release.yml`.
+2. The Registry will automatically pick up on the new version. If the registry says the release doesn't have any binaries, check the result of the `release` Github action. You might need to [Resync](https://registry.terraform.io/providers/env0/env0/latest/settings?pollNotifications=true) the registry after the action finishes.
```

Comment:
I thought the whole point of this task was so that we don't have to rely on manual re-sync, because only I have that button.

Reply:
we talk It not
https://env0.slack.com/archives/C01QBBEGBEH/p1645642100623309?thread_ts=1645603798.883379&cid=C01QBBEGBEH